pkg/store/bucket.go

```diff
@@ -42,6 +42,14 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+// Approximately this is the max number of samples that we may have in any given chunk. This is needed
+// for precalculating the number of samples that we may have to retrieve and decode for any given query
+// without downloading them. Please take a look at https://github.com/prometheus/tsdb/pull/397 to know
```

Comment:
Someone could tell that's too verbose, but I actually like this :+1:

Reply:
:+1: someone will definitely start wondering why 120 is here just like I have at the beginning